### PR TITLE
Soft delete with paranoia

### PR DESCRIPTION
**Pull request by ckvvanle**
cancancan

### CODE REVIEW

**Comment by ckvvanle** [comment]
ready

File: db/migrate/20180920040149_add_deleted_at_to_order_items.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AddDeletedAtToOrderItems < ActiveRecord::Migration[5.2]
+  def change
+    add_column :order_items, :deleted_at, :datetime
+    add_index :order_items, :deleted_at
+  end
+end
```
Comment (phamvanbk11) [inline db/migrate/20180920040149_add_deleted_at_to_order_items.rb]: gộp 3 file migrate thành 1 file thôi em

**Comment by phamvanbk11** [comment]
commented

**Comment by ckvvanle** [comment]
ready

**Pull request merged**